From c2f543622e2670bac72a0d4f3806207f53590b74 Mon Sep 17 00:00:00 2001
From: "brian.zhu" <brian.zhu@amlogic.com>
Date: Wed, 13 Nov 2013 02:44:37 +0800
Subject: [PATCH 1846/5965] remove no cached flag for res buffer interface,
 this flag will be set at the user of v4l2 driver mmap

---
 drivers/media/v4l2-core/videobuf-res.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/v4l2-core/videobuf-res.c b/drivers/media/v4l2-core/videobuf-res.c
index 2f1628f4e117..fb0d20296896 100755
--- a/drivers/media/v4l2-core/videobuf-res.c
+++ b/drivers/media/v4l2-core/videobuf-res.c
@@ -247,7 +247,7 @@ static int __videobuf_mmap_mapper(struct videobuf_queue *q,
 	size = (size < mem->size) ? size : mem->size;
 
 	//vma->vm_page_prot = vm_get_page_prot(vma->vm_flags);
-	vma->vm_page_prot = pgprot_noncached(vma->vm_page_prot);
+	//vma->vm_page_prot = pgprot_noncached(vma->vm_page_prot);
 	retval = remap_pfn_range(vma, vma->vm_start,
 				 mem->phy_addr >> PAGE_SHIFT,
 				 size, vma->vm_page_prot);
-- 
2.19.0

